src/components/RegisterForm/RegisterForm.styled.ts

```diff
@@ -0,0 +1,12 @@
+import { Alert, styled } from '@mui/material';
+import { Field } from 'formik';
+
+export const InputStyled = styled(Field)(() => {
+  return {
+    width: '100%',
+  };
+});
+
+export const AlertStyled = styled(Alert)(() => {
+  return { width: '100%' };
```

Comment:
Maybe `RegisterFormInputStyled` and `RegisterFormAlertStyled`.

If we have more of these kinds of components, it will be difficult to know which one you want to import

Reply:
changed in the latest commit.